### ISSUE TITLE
Orange 3.33.0 packages for Windows lack xgboost and catboost

### ISSUE THREAD

**Issue by pepeto** [biolab/orange3#6167]
I have just installed the new version and I found XGBoost and CatBoost grayed. I have installed them manually (pip install xgboost and pip install catboost) in the environment and CATBoost have appeared but XGBoost, have not.

![image](https://user-images.githubusercontent.com/6829629/194905749-6b32b2ef-ff1e-4c05-8e0b-b99fb42d62b4.png)

I used to have them enabled in previous installations. 

Please help. Thank you

- Operating system: Windows 10
- Orange version: 3.33 the latest
- How you installed Orange: Zip in directory

**Comment by pepeto** [comment]
I tried again to install xgboost and finally worked out. Nevertheless,this is the first time I had to install manually a library. Is there any issue with the zip version?

**Comment by markotoplak** [comment]
There might be an issue. I though xgboost should come with the Windows download (.zip or installer should both behave alike). Reopening so it gets tested.

**Comment by markotoplak** [comment]
I confirm the underlying issue: catboost and xgboost are grayed out if I unzip and use the portable Windows (`.zip`) download.

Then I looked into this more and noticed that our latest conda installers do not package xgboost anymore.

And it was all my fault. I made a mistake when building the installer. The fix for my error is this:
```
-  BUILD_LOCAL: 0
+  BUILD_LOCAL:
```

Due to my error, the Orange package was built locally and did not use the package from conda-forge. Therefore, the package did not depend on xgboost and catboost (they are conda-forge deps), so they did not get installed.

We need new installers.
